reference/shipping_provider.yml

```diff
@@ -1389,27 +1325,18 @@ components:
           description: The value associated with the product or product variant meta field
         namespace:
           type: string
-          description: The namespace associated with
-            product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield)
-            or product variant
-            (/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield)
-            meta fields. The meta field namespace should be saved under this
-            format `shipping_carrier_{yourCarrierId}` otherwise it will not be
-            recognized as an attribute.
+          description: 'The namespace associated with product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield) or product variant (/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield) meta fields. The meta field namespace should be saved under this format `shipping_carrier_{yourCarrierId}` otherwise it will not be recognized as an attribute.'
```

Comment:
```suggestion
          description: 'The namespace associated with meta fields for [products](/api-reference/catalog/catalog-api/product-metafields/createproductmetafield) and [product variants](/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield). Save the meta field namespace using the format `shipping_carrier_{yourCarrierId}`; otherwise, it will not be recognized as a valid attribute.'
```